exporter/exporterhelper/obsexporter.go

```diff
@@ -313,3 +314,34 @@ func (or *ObsReport) recordEnqueueFailureWithOtel(ctx context.Context, dataType
 
 	enqueueFailedMeasure.Add(ctx, failed, metric.WithAttributes(or.otelAttrs...))
 }
+
+func withoutCancel(parent context.Context) context.Context {
```

Comment:
Duplicate of https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/queue_sender.go#L236

Why not using that?

Reply:
good catch... i didn't see it there